### DISCUSSION TITLE
Escape as key binding

### DISCUSSION THREAD

**Discussion by biosaat** [zim-desktop-wiki/zim-desktop-wiki#1720] [Q&A]
Is it possible to use the escape key as a keybinding? 
The settings dialog does not allow this at this point in time: when pressing escape, the key input is cancelled. Is there a work-around for this? 

Background: I would like to navigate faster through my wiki. After following a link by mouse click, I want to return quickly to the previous side by just pressing escape on the keyboard.

**Comment by ZtzEAcRsND** [comment]
Assuming the 'following a link' is a zim page link, isn't this capability (go back to previous) accomplished through ```<alt><right>``` or ```<alt><left>```?  See zim help "Key BIndings".

**Reply by biosaat** [reply]
> isn't this capability (go back to previous) accomplished through `<alt><right>` or `<alt><left>`? See zim help "Key BIndings".

Yeah, this is exactly the keybinding I want to change to Esc.
